lpcxpresso55s69_cpu0 samples/subsys/usb/dfu/sample.usb.dfu build failure

**Describe the bug**

Build failure for:

`west build -p -b lpcxpresso55s69_cpu0 samples/subsys/usb/dfu`

fails with

```
/zephyrproject/bootloader/mcuboot/boot/bootutil/zephyr/../include/bootutil/bootutil_public.h:86:1: error: static assertion failed: "Unsupported value for MCUBOOT_BOOT_MAX_ALIGN"
   86 | _Static_assert(MCUBOOT_BOOT_MAX_ALIGN >= 8 && MCUBOOT_BOOT_MAX_ALIGN <= 32,
      | ^~~~~~~~~~~~~~
```

The check was introduced in

95292e85aa manifest: mcuboot update

@nvlsianpu can you take a look?

This is the only error in the last full run https://github.com/zephyrproject-rtos/zephyr/runs/8482669051

**To Reproduce**
`west build -p -b lpcxpresso55s69_cpu0 samples/subsys/usb/dfu` on current main.

**Expected behavior**
Builds

**Impact**
CI failure

--- a/samples/subsys/usb/dfu/sample.yaml
+++ b/samples/subsys/usb/dfu/sample.yaml
@@ -6,7 +6,7 @@ common:
   platform_exclude: native_posix native_posix_64 mimxrt1010_evk
     mimxrt1050_evk_qspi mimxrt1020_evk mimxrt1015_evk mimxrt1060_evk sam4l_ek
     mimxrt1050_evk mimxrt1060_evk_hyperflash nucleo_f207zg teensy40 teensy41
-    b_u585i_iot02a frdm_kl25z
+    b_u585i_iot02a frdm_kl25z lpcxpresso55s69_cpu0
   depends_on: usb_device
   filter: dt_label_with_parent_compat_enabled("slot0_partition", "fixed-partitions") and
           dt_label_with_parent_compat_enabled("slot1_partition", "fixed-partitions") and

